Description: Stop listing all 64bit architectures
 Check __LP64__ instead of maintaining a list of all
 64bit architectures.
Author: Adrian Bunk <bunk@debian.org>

--- acpica-unix-20170831.orig/source/include/platform/aclinux.h
+++ acpica-unix-20170831/source/include/platform/aclinux.h
@@ -208,9 +208,7 @@
 #define ACPI_FLUSH_CPU_CACHE()
 #define ACPI_CAST_PTHREAD_T(Pthread) ((ACPI_THREAD_ID) (Pthread))
 
-#if defined(__ia64__)    || defined(__x86_64__) ||\
-    defined(__aarch64__) || defined(__PPC64__) ||\
-    defined(__s390x__)
+#if defined(__LP64__)
 #define ACPI_MACHINE_WIDTH          64
 #define COMPILER_DEPENDENT_INT64    long
 #define COMPILER_DEPENDENT_UINT64   unsigned long
